### ISSUE TITLE
To Read & Watch

### ISSUE THREAD

**Issue by pharringtonp19** [pharringtonp19/rfp#36]
- [ ] [Neural Networks, Types, and Functional Programming](https://colah.github.io/posts/2015-09-NN-Types-FP/)
- [ ] [The Challenge of Compositionally for AI](https://compositionalintelligence.github.io/)
- [ ] [Backprop as Functors](https://www.youtube.com/watch?v=N9zZeACcV98&list=PLFE-LjWAAP9Q74cGaUF3yqUhqo2kOYY20&index=16)
- [ ] [Categorical Foundations of Gradient-Based Learning](https://arxiv.org/abs/2103.01931)
- [ ] [Compositional Game Theory](https://www.youtube.com/watch?v=5Qny8YmLUzk)
- [ ] [Seven Sketches in Compositionality](https://www.youtube.com/watch?v=w17qRHYWmDc)
- [ ] [AI Category Theory Class](https://www.youtube.com/watch?v=4poHENv4kR0&list=PLSdFiFTAI4sQ0Rg4BIZcNnU-45I9DI-VB)
- [ ] [Look at](https://www.sscardapane.it/teaching/nnds-2022/)
